src/models/indexable-meta.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Model for the SEO Meta table.
+ *
+ * @package Yoast\YoastSEO\Models
+ */
+
+namespace Yoast\YoastSEO\Models;
+
+use Yoast\YoastSEO\Yoast_Model;
+
+/**
+ * Table definition for the Indexable Meta table.
+ *
+ * @property int    $indexable_id
+ * @property string $meta_key
+ * @property string $meta_value
+ */
+class Indexable_Meta extends Yoast_Model {
```

Comment:
I would also add static functions to find all indexable_meta for an indexable, regardless of meta_key.